### ISSUE TITLE
Allow specifying a trigger/effect blocks scope somehow through cwt configs if it cannot be inferred 

### ISSUE THREAD

**Issue by adriangaro** [DragonKnightOfBreeze/Paradox-Language-Support#69]
Usually i define trigger blocks as below

```
## context_key = inline_script@jobs/resources/tec_robotic_assembly_upkeep
## context_configs_type = multiple
trigger = {
    alias_name[trigger] = alias_match_left[trigger]
}
```

Would it be possible to use `push_scope` or `replace_scopes` in the case of `context_configs_type = multiple` to provide scope type? Atm the definition works, but scope is probably any, as no type errors happen in case you use a different scope than the expected one.

**Comment by DragonKnightOfBreeze** [comment]
For parameters?

**Comment by adriangaro** [comment]
Yes this is for parameters, although would be nice to be able to somehow declare it for scripted triggers and effects too, using `definitions` in cwt for scripted triggers does not seem to work to be able to use `push_scope` `replace_scopes`

**Comment by DragonKnightOfBreeze** [comment]
Will be implemented in 1.3.5
